build_projects/dotnet-cli-build/UpdateVersionsRepo.cs

```diff
@@ -3,9 +3,12 @@
 
 using Microsoft.Build.Framework;
 using Microsoft.Build.Utilities;
-using Microsoft.DotNet.VersionTools.Automation;
 using System.IO;
 
+#if !SOURCE_BUILD
```

Comment:
Would it be better to just #if out this whole task, and then not execute the task as well?  That's the approach used elsewhere.

Reply:
Fixing. The patches did it this way and then I adjusted the others without fixing this one.